<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriftIndexFrame</class>
 <widget class="QFrame" name="DriftIndexFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font:10pt;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="7">
    <widget class="QDoubleSpinBox" name="dbReferIndex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbRange">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <item>
      <property name="text">
       <string>低量程</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中量程</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="dbConc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lbIgnoreTimes">
     <property name="text">
      <string>省略次数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lbIndex">
     <property name="text">
      <string>指标</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QDoubleSpinBox" name="dbIndex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QComboBox" name="cbSelExec">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>不执行</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>执行</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbTimes">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>重复次数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbRange">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>量程</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spTimes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cbPipe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>标样1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>标样2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>水样</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbConc">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>浓度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="lbSelExec">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>执行选择</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbPipe">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>取液管道</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spIgnoreTimes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="lbReferIndex">
     <property name="text">
      <string>参考指标</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="lbReferIndex1">
     <property name="text">
      <string>参考指标1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QDoubleSpinBox" name="dbReferIndex1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
